mackup/appsdb.py

```diff
@@ -139,7 +139,7 @@ def get_files(self, name):
         Returns:
             set of str.
         """
-        return sorted(self.apps[name]['configuration_files'])
+        return set(sorted(self.apps[name]['configuration_files']))
```

Comment:
The problem is that a set is not sorted by definition, so we'd rather remove the `sorted` until we use sorted set =/

Reply:
Make sense. 👍